### PR TITLE
fix(p-meetings): log circular objects

### PR DESCRIPTION
**Pull request by mwegman**
# Pull Request Template

## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)
[SPARK-154227](https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-154227) #code-complete

## Type of Change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Test Coverage

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:
* SDK Version
* Node/Browser Version
* NPM Version

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by mwegman** [comment]
It also strips out 'auth' headers. I don't know if we need to do that when we can just use inspect to solve the problem.

File: packages/node_modules/@webex/plugin-meetings/src/common/events/util.js [hunk 1]
```diff
@@ -6,10 +8,10 @@ EventsUtil.getEventLog = (args) => {
   let argString = '';
 
   try {
-    argString = JSON.stringify(args);
+    argString = inspect(args);
   }
   catch (e) {
-    LoggerProxy.logger.info(`Events:util#getEventLog --> ${e}`);
+    LoggerProxy.logger.warn(`Events:util#getEventLog --> ${e}`);
```
Comment (adamweeks) [inline packages/node_modules/@webex/plugin-meetings/src/common/events/util.js:14]: I wasn't sure what this was actually for. There doesn't seem to be any info as to why we would print this out or what a user could do. I guess warn is fine, but what are we warning?

Comment (mwegman) [reply packages/node_modules/@webex/plugin-meetings/src/common/events/util.js:14]: We are warning that there is a bug in our code. The user would need to submit an issue with the message in it.

Comment (adamweeks) [reply packages/node_modules/@webex/plugin-meetings/src/common/events/util.js:14]: Sounds good!

**Pull request merged**